.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - DOCKER_VERSION=1.12.6 ENABLE_SWARM=1 UPLOAD_SONATYPE=1
     - DOCKER_VERSION=1.13.1 ENABLE_SWARM=1
     - DOCKER_VERSION=17.03.0~ce ENABLE_SWARM=1
+    - DOCKER_VERSION=17.04.0~ce ENABLE_SWARM=1
```

Comment:
I want to throw this idea out for consideration: The Docker team is moving to a much more rapid release schedule, with a major `yy.mm.0` release every month and possibly minor version releases as well. So should we continue the policy to only keep the latest six major releases, or should we keep more? Is it a good idea to only test for compatibility with the releases from the last six months, rather than the years worth of releases we have now?